packages/clay-card/src/CardWithNavigation.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayCard from './Card';
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+
+interface IProps {
+	description?: React.ReactText;
+	horizontal?: boolean;
+	href?: string;
+	onClick?: () => void;
+	spritemap?: string;
+	title: React.ReactText;
+}
+
+export const ClayCardWithNavigation: React.FunctionComponent<IProps> = ({
+	children,
+	description,
+	horizontal = false,
+	href,
+	onClick,
+	spritemap,
+	title,
+}) => {
+	return (
+		<ClayCard
```

Comment:
I think we'll have to add a `tabindex="0"` in the anchor so that css does not remove the color in `horizontal` mode, when it does not have, it's leaving the `inherit` color.

Reply:
Should it always be there or only when href is present?

Reply:
And by href present, I mean when its an anchor element.

Reply:
I think that only when have an anchor can we add a `tabindex`.